reports/glides_validator.livemd

```diff
@@ -0,0 +1,154 @@
+# Glides Prediction Accuracy
+
+```elixir
+Mix.install(
+  [
+    {:csv, "~> 3.2"},
+    {:kino, "~> 0.12.0"},
+    {:ex_aws, "~> 2.5"},
+    {:ex_aws_s3, "~> 2.5"},
+    {:hackney, "~> 1.20"},
+    {:sweet_xml, "~> 0.7.4"},
+    {:timex, "~> 3.7"},
+    {:jaxon, "~> 2.0"}
+  ],
+  config: [
+    ex_aws: [
+      access_key_id: System.get_env("LB_AWS_ACCESS_KEY_ID"),
```

Comment:
Sure! I'll add some documentation for how to set this up. It would be necessary for someone to have those to pull from the S3 archive. In LiveBook you can add secrets through the UI, and they get prefixed with LB.